ow_power_system/src/power_system_node.cpp

```diff
@@ -17,27 +24,112 @@ int main(int argc, char* argv[]) {
 
   //Construct our power draw listener
   //TODO: Check out latching
-  ros::Subscriber power_listener = n.subscribe("power_draw", 1000, computeSOC);
+  //ros::Subscriber power_listener = n.subscribe("power_draw", 1000, computeSOC);
 
-  //Construct our State of Charge (SOC) publisher
+  //Construct our State of Charge (SOC) publisher (Volts)
   ros::Publisher SOC_pub = n.advertise<std_msgs::Float64>("state_of_charge",1000);
+  //Construct our Remaining Useful Life (RUL) publisher (Seconds)
+  ros::Publisher RUL_pub = n.advertise<std_msgs::Float64>("remaining_useful_life",1000);
+  
+  //Load power values csv
+  string csv_path,csv_file;
+  csv_file = "/data/onewatt.csv";
+  //FILE OPTIONS: onewatt.csv, eightwatt.csv, sixteenwatt.csv
+  csv_path = ros::package::getPath("ow_power_system");
+  csv_path += csv_file;
+
+  auto power_csv = loadCSV(csv_path);
+  cout << "Power system CSV loaded!";
```

Comment:
Please remove any C++ stream output messages that are just for your development or debugging.  If the message is important for users (probably not this one), use ROS output (ROS_INFO, ROS_WARN, ROS_ERROR).   You can also use ROS_DEBUG to keep your debugging messages in.

Reply:
Done.